static-assets/scripts/resources/locale-de.json

```diff
@@ -272,32 +273,32 @@
         "SUCCESSFULLY_PUSHED": "Push war erfolgreich",
         "SUCCESSFULLY_PULLED": "Pull war erfolgreich",
         "REPO_DELETED": "Repository wurde gelöscht",
-        "MERGE_STRATEGY_LABEL": "Strategie zusammenführen",
+        "MERGE_STRATEGY_LABEL": "Merge-Strategie",
         "MERGE_STRATEGIES": {
-          "none": "Keiner",
-          "ours": "Akzeptiere unsere",
-          "theirs": "Akzeptiere ihre"
+          "none": "Keine",
+          "ours": "OURS",
```

Comment:
Is it generally used and better understood as just “ours” / “theirs” in German instead of accept ours / accept theirs? Is the all capitals intentional?

Reply:
Hmm, there is no good translation into german. When it comes to git-related terms, I would keep them in english.

Reply:
Makes sense.

Did you write ”OURS” and “THEIRS” all in capitals on purpose? Instead of regular capitalization (i.e. Ours, Theirs)

Reply:
I've changed them to lowercase